### PR TITLE
New spanish dubbed site PelisplusHD

### PR DESCRIPTION
**Pull request by Stormunblessed**
This is a popular site pelisplushd.net (it's not a mirror of pelisplus.icu), almost 90% of its library are spanish dubs with good quality.

### CODE REVIEW

**Comment by LagradOst** [comment]
LGTM

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusHDProvider.kt [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        }
+
+        if (items.size <= 0) throw ErrorLoadingException()
+        return HomePageResponse(items)
+    }
+
+    override fun search(query: String): List<SearchResponse> {
+        val url = "https://pelisplushd.net/search?s=${query}"
+        val html = app.get(url).text
+        val document = Jsoup.parse(html)
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusHDProvider.kt:61]: be aware that you can use val document = app.get(url).document

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusHDProvider.kt [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    override fun loadLinks(
+        data: String,
+        isCasting: Boolean,
+        subtitleCallback: (SubtitleFile) -> Unit,
+        callback: (ExtractorLink) -> Unit
+    ): Boolean {
+        val html = app.get(data).text
+        val soup = Jsoup.parse(html)
+        val selector = soup.selectFirst("div.player > script").toString()
+        val linkRegex = Regex("""(https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/\/=]*))""")
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusHDProvider.kt:158]: Sick regex, but isn't it needlessly complicated? You can probably get away with using .* at places. No need to change.

**Review by Blatzar — APPROVED** [review]
looks good

**Pull request merged**